server/src/application/application.service.ts

```diff
@@ -71,4 +74,19 @@ export class ApplicationService {
   ): Promise<void> {
     await this.applicationRepository.update(applicationId, applicationDto);
   }
+
+  async assignToUser(applicationId: string, assignToUserDto: AssignToUserDto): Promise<void> {
+    const application = await this.getApplication(applicationId);
+    if (application) {
```

Comment:
Should throw an error if application is not there! I'll add it somewhere on how that needs to be done!

Reply:
Would it be something like:
```
if (!application) throw EntityNotFoundError;
```
?